sdks/python/container/run_validatescontainer.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# This script will be run by Jenkins as a post commit test. In order to run
+# locally make the following changes:
+#
+# LOCAL_PATH   -> Path of tox and virtualenv if you have them already installed.
+# GCS_LOCATION -> Temporary location to use for service tests.
+# PROJECT      -> Project name to use for dataflow and docker images.
+#
+# Execute from the root of the repository: sdks/python/container/run_validatescontainer.sh
+
+set -e
+set -v
+
+# pip install --user installation location.
+LOCAL_PATH=$HOME/.local/bin/
+
+# Where to store integration test outputs.
+GCS_LOCATION=gs://temp-storage-for-end-to-end-tests
+
+# Project for the container and integration test
+PROJECT=apache-beam-testing
+
+# Verify in the root of the repository
+test -d sdks/python/container
+
+# Verify docker and gcloud commands exist
+command -v docker
+command -v gcloud
+docker -v
+gcloud -v
+
+# ensure maven version is 3.5 or above
+TMPDIR=$(mktemp -d)
+MVN=$(which mvn)
+mvn_ver=$($MVN -v | head -1 | awk '{print $3}')
+if [[ "$mvn_ver" < "3.5" ]]
+then
+  pushd $TMPDIR
+  curl http://www.apache.org/dist/maven/maven-3/3.5.2/binaries/apache-maven-3.5.2-bin.tar.gz --output maven.tar.gz
+  tar xf maven.tar.gz
+  MVN="$(pwd)/apache-maven-3.5.2/bin/mvn"
+  popd
+fi
+
+# ensure gcloud is version 186 or above
+gcloud_ver=$(gcloud -v | head -1 | awk '{print $4}')
+if [[ "$gcloud_ver" < "186" ]]
+then
+  pushd $TMPDIR
+  curl https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-186.0.0-linux-x86_64.tar.gz --output gcloud.tar.gz
+  tar xf gcloud.tar.gz
+  ./google-cloud-sdk/install.sh --quiet
+  . ./google-cloud-sdk/path.bash.inc
+  popd
+  gcloud components update --quiet || echo 'gcloud components update failed'
+  gcloud -v
+fi
+
+# Build the container
+TAG=$(date +%Y%m%d-%H%M%S)
+CONTAINER=us.gcr.io/$PROJECT/$USER/python:$TAG
+echo "Using container $CONTAINER"
+$MVN clean install -DskipTests -Pbuild-containers --projects sdks/python/container -Ddocker-repository-root=us.gcr.io/$PROJECT/$USER -Ddockerfile.tag=$TAG -amd
```

Comment:
Can this be built with gradle yet?

Reply:
Not yet? sdks/python/container/build.gradle doesn't allow setting the image name, and ./gradlew :sdks:python:container:docker fails for me? @herohde

Reply:
@kennknowles Can this PR be checked in without converting the container build to gradle?

Reply:
I haven't had a chance to review it again but @aaltay and @herohde are enough. You'll need to squash the commits, I expect.

Reply:
Great. Squashed the commits into one.